ci/local/build.sh

```diff
@@ -1,6 +1,9 @@
 #!/bin/bash
 
-DOCKER_IMAGE="gpuci/rapidsai-base:cuda10.0-ubuntu16.04-gcc5-py3.6"
+GIT_DESCRIBE_TAG=`git describe --tags`
+MINOR_VERSION=`echo $GIT_DESCRIBE_TAG | grep -o -E '([0-9]+\.[0-9]+)'`
+
+DOCKER_IMAGE="gpuci/rapidsai:${MINOR_VERSION}-cuda10.1-devel-ubuntu16.04-py3.6"
```

Comment:
looks good Sean. In the PR below, we'll be removing Python 3.6 support though. So it's probably a good idea to change this image to end with `py3.7` instead of `py3.6`.

https://github.com/rapidsai/gpuci-build-environment/pull/104

Reply:
Thanks for catching that, AJ! Change has been added to this PR. Will make the change to the other PRs as well.